app/frontend/src/AppRoutes.tsx

```diff
@@ -136,12 +138,15 @@ export default function AppRoutes() {
       {
         // PROFILE
       }
-      <AppRoute isPrivate path={editUserRoute}>
+      <AppRoute isPrivate path={editProfileRoute}>
         <EditProfilePage />
       </AppRoute>
-      <AppRoute variant="full-width" isPrivate path={userRoute}>
+      <AppRoute isPrivate path={profileRoute}>
```

Comment:
```suggestion
      <AppRoute variant="full-width" isPrivate path={profileRoute}>
```
So it's consistent with the user page - it's mainly there so the cards stretches the full width of the screen when on breakpoints below `md`.